src/modules/project/pages/projectFunding/views/FundingComplete.tsx

```diff
@@ -2,21 +2,25 @@ import { Box, Button, Text, VStack } from '@chakra-ui/react'
 import { useTranslation } from 'react-i18next'
 import { FaCheck } from 'react-icons/fa'
 
-import { ProjectFragment } from '../../../../../types'
+import { Grant, ProjectFragment, VotingSystem } from '../../../../../types'
 import { ProjectFundingFormState } from './FundingForm'
 
 interface Props {
   formState: ProjectFundingFormState | undefined
   project: ProjectFragment
+  grant?: Grant
+  onClose: () => void
 }
 
-export const FundingComplete = ({ formState, project }: Props) => {
+export const FundingComplete = ({ formState, project, grant, onClose }: Props) => {
   const { t } = useTranslation()
 
-  const handleClose = () => {
-    window.location.reload()
+  const handleCopyLink = () => {
+    navigator.clipboard.writeText(window.location.href)
   }
 
+  const voteCount = Math.floor(Math.log10(formState?.donationAmount || 0)) - 2
```

Comment:
A bit of tech debt here because it's not looking at the Voting System type. No need to address it now, but I'd suggest adding that as a comment above that line for future ref.